### ISSUE TITLE
Make it available via package managers

### ISSUE THREAD

**Issue by o-sdn-o** [netxs-group/VTM#77]
Package managers

 - [ ] AUR Arch Linux
 - [ ] PPA
 - [ ] Homebrew
 - [ ] MacPorts
 - [ ] zypper OpenSUSE Linux distribution
 - [ ] scoop on Windows
 - [ ] choco, Chocolatey for Windows

**Comment by UltraStudioLTD** [comment]
**Termux** (**Android**) `apt` and `pkg` (wrapper over `apt`) ✅

**Comment by Vosjedev** [comment]
When the vtm store is done, it should also be able to update vtm without the need of a system package manager. This means when vtm is not on the system package manager, vtm could update itself. This also means that the store has to see if the system packager can update vtm, or if it has to update itself.

**Comment by o-sdn-o** [comment]
I plan to support such a scenario in the future. Currently it is iceboxed #337 until the store is done.

**Comment by megatog615** [comment]
How about Debian?

**Comment by o-sdn-o** [comment]
Indeed, if I understood everything correctly, I need to make a Request for Package (Debian). I've added this to the todo-list. Thank you.

**Comment by o-sdn-o** [comment]
TIL https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=898259
